packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -81,6 +83,14 @@ def hermes_engine_tarball_envvar_defined()
     return ENV.has_key?('HERMES_ENGINE_TARBALL_PATH')
 end
 
+def react_native_maven_local_path_envvar_defined()
+    return ENV.has_key?('REACT_NATIVE_MAVEN_LOCAL_PATH')
+end
+
+def react_native_maven_namespace()
+    return 'com.facebook.react'
+end
+
```

Comment:
why do we need this? iOS uses a different artifact for Hermes, we should not use the one built in Maven local

Reply:
The CI I have built for RNTV works differently, and does use iOS artifacts in Maven local. So I do find these changes useful. However, I am happy to drop them this PR for now.

Reply:
Fixed